### PR TITLE
i have updated the corrected images paths

### PR DESCRIPTION
**Pull request by khadijahMu**

### CODE REVIEW

**Comment by Xfarouq** [comment]
@khadijahMu why are your files empty??

**Comment by khadijahMu** [comment]
let me check them

**Comment by khadijahMu** [comment]
i have added my files

**Review by Devsalmasaeed — CHANGES_REQUESTED** [review]
Hi @khadijahMu,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️
- All repetitive content of the webpage should be rendered dynamically using JavaScript (Why Choose Us, Special Offers, Reviews and Recent Posts)
- Deploy your website on render and update your README

 

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Comment by khadijahMu** [comment]
1. I have changed all repetitive contents and rendered them dynamically 
2. i have updated my Readme and redployed my site on render

**Review by Xfarouq — CHANGES_REQUESTED** [review]
Hi @khadijahMu 

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️
- It's best practice to always not leave spaces in your code, so please remove them and make sure your code is well indented
- put all your images folder in a folder named assests  for best practices
- Remove unused css files which isn't used for the code
- Make your menu bar hidden unless when the hamburger menu is clicked for it to pop up
- Remove your html codes for the sections which you rendered using javascript

- kindly consider having the app's name on the splash screen as requested in the project requirement or as given on the design

### Optional Suggestions
- You can make your review section to stop showing the horixontal bar by doing overflow-x: scroll

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Comment by khadijahMu** [comment]
I have made all the required changes

**Comment by Xfarouq** [comment]
Ok you can submit again for review

File: main.js [hunk 1]
```diff
@@ -1,44 +1,226 @@
...
-    const additionalReviews = document.querySelector('.additional-reviews');
-    
-    // Check if the reviews are currently visible
-    if (additionalReviews.style.display === 'none' || additionalReviews.style.display === '') {
-        additionalReviews.style.display = 'block';  // Show the reviews
-    } else {
-        additionalReviews.style.display = 'none';  // Hide the reviews
-    }
-});
-
```
Comment (Xfarouq) [inline main.js:11]: avoid leaving whitespaces please

File: main.js [hunk 2]
```diff
@@ -1,44 +1,226 @@
...
-// Show the menu page when the menu icon is clicked
-menuIcon.addEventListener('click', function() {
-    menuPage.style.display = 'block';  // Display the menu page
-});
-
-// Hide the menu page when the close icon is clicked
-closeMenu.addEventListener('click', function() {
-    menuPage.style.display = 'none';  // Hide the menu page
-});
-
```
Comment (Xfarouq) [inline main.js:27]: avoid leaving spaces

File: index.html [hunk 3]
```diff
@@ -1,726 +1,506 @@
...
-                <p class="offer-title">Wilderness Club at Big Cedar</p>
-                <p class="offer-date-price">28 October — 1 November</p>
-                <span class="price">
-                    <span class="price-large">$2016</span> 
-                    <span class="price-small">/ 6 nights</span>
-                </span>
-            </div>
-        </div>
-                <!-- fourth deal -->
-        <div class="special-offer">
```
Comment (Xfarouq) [inline index.html:194]: Please remove all html used for the rendered sections just leave the ID name used to call the rendered sections in javascript

**Review by Xfarouq — CHANGES_REQUESTED** [review]
Hi @khadijahMu 

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️

- You still haven't implemented previous changes
- Your images should all be in one folder called images and then put the images folder in assets folder
- If your having any difficulty or you don't understand anything please your always free to reach out and ask
- kindly consider having the app's name on the splash screen as requested in the project requirement or as given on the design

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Comment by khadijahMu** [comment]
1- i have added the project name as splash screen
2- my last deal section was not rendered dynamically, but i have deleted those that were dynamically rendered (review,why choose us, and special offers).
3- i have created the assets folder.
4-i have removed lines and white spaces.

**Pull request merged**